src/App.js

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import "./styles.css";
+import Stars from "./Stars";
+
+const sections = {
+  ALL: "All",
+  CORE: "Core Framework",
+  PLUGIN: "Plugins",
```

Comment:
should add "community" tag, but probably kept under plugins :thinking:

Reply:
can add a new section, is there any existing community doc to add already?